support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -117,24 +115,13 @@ class SupportWorkersClient(
     product match {
       case _: GuardianWeekly =>
         Right(
-          GiftRecipient.WeeklyGiftRecipient(
```

Comment:
There used to be two types of gift recipient - one for Digital Subscription giftees and one for Guardian Weekly giftees. Now Guardian Weekly is the only product which still supports gifting, so we only need one case class